pkg/tcpip/network/fragmentation/fragmentation.go

```diff
@@ -22,9 +22,14 @@ import (
 	"sync"
 	"time"
 
+	"gvisor.dev/gvisor/pkg/tcpip"
 	"gvisor.dev/gvisor/pkg/tcpip/buffer"
+	"gvisor.dev/gvisor/pkg/tcpip/header"
+	"gvisor.dev/gvisor/pkg/tcpip/stack"
 )
 
+var R *stack.Route
```

Comment:
why?

Reply:
Earlier, we thought of passing 'r' as parameter to Process. But this was resulting in failure of some test cases in fragmentation_test.go
Also, it involved additional effort to fix those test cases. So, we decided to introduce a new variable 'R'.

Reply:
Sorry the correct approach is to pass an extra parameter as required. Please get rid of this global